python/tests/hypothesis/arcticdb/test_hypothesis_version_store.py

```diff
@@ -114,7 +114,7 @@ def _get_latest_undeleted_version(self, sym) -> Tuple[Optional[int], Optional[Ve
 
     @rule(
         sym=symbols,
-        prune=st.sampled_from([True, False]),  # booleans() shrinks towards False, but prune is on by default
+        prune=st.sampled_from([True, False]),  # booleans() shrinks towards False, but prune is off by default
```

Comment:
I think that just 

```
# booleans() shrinks towards False
```

is clearer, the default prune_previous isn't used by the logic of the test so the comment was misleading before

Reply:
removed